src/dlio_postprocessor.py

```diff
@@ -49,17 +50,17 @@ def __init__(self, args) -> None:
         self.verify_and_load_all_files()
         self.disks = []
         self.overall_stats = {}
-
+        self.record_size = args.record_size
```

Comment:
Why do we keep copying args into every class. It increases the memory usage unnecessary. Please use refer to args singleton and use that directly.

Reply:
Because args is only available in __init__

Reply:
I mean we should dont need to pass the args at all, we can get the singleton here and use that directly as a class variable.